beacon/goclient/singing.go

```diff
@@ -9,8 +9,21 @@ import (
 	"github.com/prysmaticlabs/go-ssz"
 )
 
+// SigningRoot is a wrapping type for byte array
+type SigningRoot [32]byte
```

Comment:
@amir-blox why do we need this?

Reply:
@alonmuroch in order to identify an empty array (panic happens because signing root is empty)